lldb/test/API/functionalities/data-formatter/data-formatter-stl/libstdcpp/string/TestDataFormatterStdString.py

```diff
@@ -17,6 +17,15 @@ def setUp(self):
         # Find the line number to break at.
         self.line = line_number("main.cpp", "// Set break point at this line.")
 
+    # This is the function to remove the custom formats in order to have a
```

Comment:
If you make all your custom summaries in a category (e.g. pass `-w SOME_UNIQUE_NAME` then you can delete just the category with `type category delete`.  That way you know you are only deleting what you added.

Categories come into being disabled, so if you do it this way, you need to add:

`type category enable SOME_UNIQUE_NAME`

to your `__lldb_init_module`

Reply:
Ah, I see you didn't originally do that, so don't feel required to do that, but it's still the better way to do it...

Reply:
This PR is already old enough that we can lump it in!